indigo_app/views/tasks.py

```diff
@@ -32,7 +33,25 @@ class TaskListView(TaskViewBase, ListView):
     paginate_orphans = 4
 
     def get_queryset(self):
-        return Task.objects.filter(country=self.country, locality=self.locality).order_by('-created_at')
+        try:
+            frbr_uri = self.request.GET['frbr_uri']
+        except MultiValueDictKeyError:
+            frbr_uri = None
+        if frbr_uri:
+            return Task.objects.filter(work__frbr_uri=frbr_uri).order_by('-created_at')
```

Comment:
We should first filter by country and locality. Otherwise, I can go to `/places/za-cpt/tasks?frbr_uri=/na/act/2014/1` and see tasks for a Namibian work on the Cape Town task list.

So filtering by the `frbr_uri` should happen *in addition* to the place filtering.